code/__defines/dcs/signals.dm

```diff
@@ -84,4 +84,7 @@
 /// updates sprites
 #define COMSIG_ITEM_ICON_UPDATE "icon_update"
 
+// tgui signals
+#define COMSIG_TGUI_CLOSE "tgui_close"
```

Comment:
Does TG not have a signal or method to handle this already? It would strike me as odd if they do not

Reply:
i know right? maybe we just didn't port it, but there isn't any signals for that. there's only one for UI_ACT